sdk/src/main/java/com/vk/api/sdk/queries/groups/GroupsSetCallbackSettingsQuery.java

```diff
@@ -406,8 +478,132 @@ public GroupsSetCallbackSettingsQuery groupChangePhoto(Boolean value) {
+     *
+     * @param flags flags which should be added (set to {@code true})
+     * @return a reference to this {@code AbstractQueryBuilder} object to fulfill the "Builder" pattern.
+     */
+    public GroupsSetCallbackSettingsQuery addFlags(CallbackFlag... flags) {
+        this.flags.addAll(Arrays.asList(flags));
+        return updateUnsafeParamsWithFlags();
+    }
+
+    /**
+     * Removes all given {@code CallbackFlag}s from Set of this {@code GroupsSetCallbackSettingsQuery}`s flags
+     *
+     * @param flags flags which should be removed (set to {@code false})
+     * @return a reference to this {@code AbstractQueryBuilder} object to fulfill the "Builder" pattern.
+     */
+    public GroupsSetCallbackSettingsQuery removeFlags(CallbackFlag... flags) {
+        this.flags.removeAll(Arrays.asList(flags));
+        return updateUnsafeParamsWithFlags();
+    }
+
+    /**
+     * Sets the given {@code CallbackFlag} to the {@code Boolean} value given
+     *
+     * @param flag flag to be updated
+     * @param value value to which update the flag
+     * @return a reference to this {@code AbstractQueryBuilder} object to fulfill the "Builder" pattern.
+     */
+    public GroupsSetCallbackSettingsQuery setFlag(CallbackFlag flag, Boolean value) {
+        if (value) flags.add(flag);
+        else flags.remove(flag);
+        return unsafeParam(flag.unsafeKey, value);
+    }
+
+    /**
+     * Clears the Set of {@code CallBackFlag}s and adds all flags given
+     *
+     * @param flags flags which should be added (set to {@code true})
+     * @return a reference to this {@code AbstractQueryBuilder} object to fulfill the "Builder" pattern.
+     */
+    public GroupsSetCallbackSettingsQuery setFlags(CallbackFlag... flags) {
+        this.flags = new HashSet<>(Arrays.asList(flags));
+        return updateUnsafeParamsWithFlags();
+    }
+
+    /**
+     * Clears the Set of {@code CallBackFlag}s which is similar to setting all flags to {@code false}
+     *
+     * @return a reference to this {@code AbstractQueryBuilder} object to fulfill the "Builder" pattern.
+     */
+    public GroupsSetCallbackSettingsQuery clearFlags() {
+        this.flags.clear();
+        return updateUnsafeParamsWithFlags();
+    }
+
+    /**
+     * Gets the Set of all flags set to {@code true} for this {@code GroupsSetCallbackSettingsQuery}
+     *
+     * @return a reference to this {@code AbstractQueryBuilder} object to fulfill the "Builder" pattern.
+     */
+    public Set<CallbackFlag> getFlags() {
+        return flags;
+    }
+
+    /**
+     * Updates unsafe params for all existing flags
+     *
+     * @return a reference to this {@code AbstractQueryBuilder} object to fulfill the "Builder" pattern.
+     */
+    private GroupsSetCallbackSettingsQuery updateUnsafeParamsWithFlags() {
+        for (CallbackFlag flag : CallbackFlag.values()) unsafeParam(flag.unsafeKey, flags.contains(flag));
+        return getThis();
+    }
+
+    enum CallbackFlag {
```

Comment:
Already exists in CallbackApi.
May be need to be changed to public and reused here. That helps to prevent code dublication.